src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/LaunchProfiles/IProjectLaunchProfileHandler.cs

```diff
@@ -1,9 +1,9 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
 using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.ProjectSystem.Debug;
```

Comment:
There are two `ILaunchProfile` interfaces: the one specific to the Query API (defined by CPS) and the one coming from our code which represents a launch profile internally. I suspect `ILaunchProfile` used to bind to the former, but with this using statement probably now binds to the latter.